recipes/metabat/meta.yaml

```diff
@@ -0,0 +1,30 @@
+package:
+  name: metabat
+  version: 2.12.1
+
+build:
+  number: 0
+  skip: True  # [not linux64 or not py27]
+
+source:
+  url: https://bitbucket.org/berkeleylab/metabat/downloads/metabat-static-binary-linux-x64_v2.12.1.tar.gz
+  sha256: 1fc70cb388feac43392aa87190cd368797cb88dbd0be8b541b8019187436dbc3
+  patches:
+    - metabat.patch
+
+requirements:
+  run:
+    - python
+    - perl
+    - boost >=1.55.0,{{CONDA_BOOST}}*
+    - zlib >=1.2.4,{{ CONDA_ZLIB }}*
+
+test:
+  commands:
+    - metabat -h
+
+about:
+  home: https://bitbucket.org/berkeleylab/metabat
+  license: Free for academic, nonprofit and personal use
```

Comment:
Can you clarify the license? There doesn't appear to be a commercial restriction in the [`license.txt`](https://bitbucket.org/berkeleylab/metabat/raw/235ca312d75583514c20278278522cea291562c8/license.txt) file. The short id to use for the LBL [license](https://spdx.org/licenses/BSD-3-Clause-LBNL.html) is `  license: BSD-3-Clause-LBNL`.

Reply:
You mean I should ask the author of this tool to take the commercial use into consideration?